src/node-capnp/capnp.js

```diff
@@ -254,8 +254,6 @@ function Capability(native, schema) {
     return new Capability(v8capnp.castAs(native, newSchema), newSchema);
   }
   this.schema = schema;
-
-  Object.freeze(this);
```

Comment:
sug: Perhaps doing `Object.seal(this)` would be closer to the original intent whilst still resolving the bug?

Reply:
Good thinking! The fact that we can't be sure of the original intent makes me a little nervous, so a smaller change seems less likely to bite us in the long run.